views/whereabouts.njk

```diff
@@ -12,13 +16,76 @@
       href: "/search-prisoner-whereabouts"
     },
     {
-      text: "Dashboard"
+      text: "Statistics: attendance reasons"
     }
   ]
 }) 
 }}
 {% endblock %}
 
 {% block content %}
-Whereabouts Dashboard goes here...
+
+{% set urlParams = '/whereabouts?agencyId=' + caseLoadId + '&period=AM&date=2019-08-23'%}
+
+<h1 class="govuk-heading-l mainHeading">Statistics: attendance reasons</h1>
```

Comment:
What does `mainHeading` do?

Reply:
Good spot.  Diddly.  It's left in from before the actual heading classes were put in.